lib/livebook_web/live/hooks/user_hook.ex

```diff
@@ -20,7 +20,7 @@ defmodule LivebookWeb.UserHook do
          {:user_change, %{id: id} = user},
          %{assigns: %{current_user: %{id: id}}} = socket
        ) do
-    {:cont, assign(socket, :current_user, user)}
+    {:halt, assign(socket, :current_user, user)}
```

Comment:
I think that Livebook.Session also matches on this event, so we need to keep it as `:cont`. :)

Reply:
In fact, please check if everything is working as expected in the user modal inside the notebook/session!

Reply:
It is, i can change the color and it broadcasts to all users in current session.
And yeah, with the `push_redirect/2`, I don't need to keep this change